spec/models/manageiq/providers/amazon/storage_manager/ebs/stubbed_refresher_spec.rb

```diff
@@ -183,4 +187,35 @@ def assert_ems
     expect(ems.cloud_volumes.size).to eql(expected_table_counts[:cloud_volume])
     expect(ems.cloud_volume_snapshots.size).to eql(expected_table_counts[:cloud_volume_snapshot])
   end
+
+  def assert_specific_volume
+    @volume = ManageIQ::Providers::Amazon::StorageManager::Ebs::CloudVolume.where(:ems_ref => "volume_id_1").first
+
+    expect(@volume).not_to be_nil
+    expect(@volume).to have_attributes(
+      :ems_ref     => "volume_id_1",
+      :name        => "volume_id_1",
+      :status      => "in-use",
+      :volume_type => "standard",
+      :size        => 1.gigabyte
+    )
+
+    expect(@volume.ext_management_system).to eq(@ems.ebs_storage_manager)
+    expect(@volume.base_snapshot).to eq(@snapshot)
+  end
+
+  def assert_specific_snapshot
```

Comment:
Move this above `assert_specific_volume`, please. It makes more sense to define functions in same order as they should be executed.